src/CalcManager/CEngine/Number.cpp

```diff
@@ -42,23 +33,14 @@ namespace CalcEngine
         return ret;
     }
 
-    int32_t const& Number::Sign() const
-    {
-        return m_sign;
-    }
+    int32_t const& Number::Sign() const { return m_sign; }
```

Comment:
What setting do we change so that this is reverted?  The previous style is the goal.

Reply:
> this is not really possible to keep the previous style at all.

Can't we use `AllowShortFunctionsOnASingleLine: None`?

Reply:
@danbelcher-MSFT I will check it, but I think I have tried it already.

As I tried to explain, such precise adjustments are tough with clang-format. 
Also, I would try custom rules for braces. If it doesn't solve the problem - will mean that it is not possible. Let's see.

Reply:
@danbelcher-MSFT fixed.